about.html

```diff
@@ -0,0 +1,103 @@
+      <nav class="navbar">
+        <div class="hamburger">
+          <img src="./img/Hamburger_icon.svg.png" alt="Menu Hamburger" />
+        </div>
+        <ul class="nav-links">
+          <li class="close-menu-btn">
+            <button type="submit" >
+              <span>&times;</span>
+            </button>
+          </li>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html" id="current">About</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="about-section">
+        <div class="about-wrapper">
+        <div class="about-section-header">
+          <p class="about-greeting">"Hello! Sharing World"</p>
+          <h1 class="about-info">Unlocking the potential of those who advance the world</h1>
+        </div>
+        <div class="about-text">
+          <p>
+            Get rid of the hassle of starting from scratch with business consulting website template, Consul. It’s a tool of many great advantages, starting with a professional design. You can create a powerful website by using Consul as-is, but that’s not necessary.
+          </p>
+        </div>
+
+        <div class="about-contact">
+          <p>Please contact us per Email for any further questions about Consul</p>
+          <a href="#" class="about-adress">
+            consul@gmail.com
+          </a>
+        </div>
+      </div>
+      </section>
+      <section class="about-details">
+        <div class="about-details-wrapper">
+          <h2 class="about-details-title">The gain insights, advice and tools to achieve your mission-critical priorities</h2>
+          
+
+          <p>The logo of Consul was decided through the logo competition from 8 june to 7 july</p>
+        </div>
+
+        <div class="about-details-img">
+          <img src="./img/170-1705916_hashicorp-consul.png" alt=" Global ">
+        </div>
+      </section>
+      
+      <div class="about-middle-page">
+        <div class="about-middle-page-wrapper">
+          <h2>See the past of Consul</h2>
+          <p>We help leading organizations succed with their most critical</p>
+        </div>
+      </div>
+
+      <div class="about-images">
+        <img src="./img/c1.jpeg" alt=" Author">
+        <img src="./img/Angela-Shante_NoisyClassroom-300x212.jpg" alt=" Author">
+        <img src="./img/c2.jpeg" alt=" Author">
+      </div>
+      <footer>
+        <div class="footer-header">
+          <h2>Partner</h2>
+        </div>
+        <ul class="footer-refs">
+          <li><a href="#"><img src="./img/logo1.png" alt=""></a></li>
+          <li><a href="#"><img src="./img/logo2.png" alt=""></a></li>
+          <li><a href="#"><img src="./img/logo5.png" alt=""></a></li>
+          <li><a href="#"><img src="./img/logo6.png" alt=""></a></li>
+          <li><a href="#"><img src="./img/logo9.png" alt=""></a></li>
+        </ul>
+        <div class="footer-bottom">
```

Comment:
- [ ] The About page should contain:
-  a top section with a centralised column that includes a page title, a text block with the page description, and a line with contact information or any other prompt under it. The contact should be a link as it is an email address. It should be bold and underlined as shown in the Behance design.
- a middle section with a centralised column that includes a section header, a divider, a paragraph of text, and an image or large logo. Reduce the font size of the header in this section.
- a bottom section with a centralised column that includes a section header, a divider, a line of text, and two images with text centred over them. 

_See images below._

**Original** | **Your project**
  ----------- | -----------
![image](https://user-images.githubusercontent.com/49621279/123981173-fbdf0f00-d976-11eb-80df-da036b8ad793.png) | ![image](https://user-images.githubusercontent.com/49621279/123981353-1f09be80-d977-11eb-8686-ba00cc7b9803.png)